chartfx-math/src/main/codegen/de/gsi/math/MathGen.java

```diff
@@ -323,7 +321,7 @@ public double KolmogorovTest(int na, double[] a, int nb, double[] b, String opti
      * @param index see above
      * @param type see above
      */
-    void Quantiles(int n, int nprob, double[] x, double[] quantiles, double[] prob, boolean isSorted, int[] index, int type) {
+    void quantiles(int n, int nprob, double[] x, double[] quantiles, double[] prob, boolean isSorted, int[] index, int type) {
```

Comment:
Codacy found an issue: [The method 'quantiles(int, int, double, double, double, boolean, int, int)' has an NPath complexity of 64638, current threshold is 200](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)